gym_hil/envs/panda_arrange_boxes_gym_env.py

```diff
@@ -0,0 +1,194 @@
+#!/usr/bin/env python
+
+import gymnasium as gym
+from typing import Any, Dict, Literal, Tuple
+from pathlib import Path
+import mujoco
+import numpy as np
+from gymnasium import spaces
+
+from gym_hil.mujoco_gym_env import FrankaGymEnv, GymRenderingSpec
+
+_PANDA_HOME = np.asarray((0, 0.195, 0, -2.43, 0, 2.62, 0.785))
```

Comment:
Could you change this home position to this, so that the scene can be nicely seen?
```suggestion
_PANDA_HOME = _PANDA_HOME = np.asarray((0, -0.785, 0, -2.35, 0, 1.57, np.pi / 4))